src/plonk_recursion.rs

```diff
@@ -30,8 +30,7 @@ pub struct RecursionPublicInputs {
     o_plonk_z_local: PublicInput,
     o_plonk_z_right: PublicInput,
     o_plonk_t: Vec<PublicInput>,
-    halo_u_l: Vec<PublicInput>,
-    halo_u_r: Vec<PublicInput>,
+    halo_us: Vec<PublicInput>,
     old_proofs: Vec<PublicInput>,
 }
 
```

Comment:
Is the `to_vec` function used somewhere?

Reply:
Oh yeah seems it's not used. Deleted it :)